Automattic/jetpack
# Publicize: Remove assets directory

These files are no longer required.

Some were no longer needed as of 3.0. Others as of 7.1. Others somewhere in between.

#### Changes proposed in this Pull Request:
* Removes the entire `modules/publicize/assets/` directory.

#### Testing instructions:
I couldn't find any reference to any of these files. Try playing around with Settings -> Sharing, and the Publicize UI in both Classic and Block editors to see if you can generate any 404s.

#### Proposed changelog entry for your changes:
* Remove files no longer required after recent Publicize changes.
